frontend/src/tests/application-tests.js

```diff
@@ -1,19 +1,116 @@
-import { it, beforeAll } from "./utils";
+import {
+    it,
+    beforeAll,
+    expect,
+    checkPropTypes,
+    postingPropTypes,
+} from "./utils";
 import { databaseSeeder } from "./setup";
 
 export function applicationsTests({ apiGET, apiPOST }) {
-    beforeAll(async () => {
-        await databaseSeeder.seed({ apiGET, apiPOST });
-    }, 30000);
+    let session;
+    let position;
+    let posting = {};
+    let resp;
```

Comment:
Define this inside each function that needs it. There are potential race conditions otherwise.

Reply:
Currently these are declared outside to be used as global variables and defined in `beforeAll` through seeders. To clarify, are you saying they should be defined from calling the seeder in each case that needs it?